### PR TITLE
Leaderboard:- Implment styles

### PR DESCRIPTION
**Pull request by Melaku05**
-adding the styles to the project.
- uses CSS to improve the look and feel of the application.

### CODE REVIEW

**Review by alexander16108 — APPROVED** [review]

File: src/style.css [hunk 1]
```diff
@@ -34,13 +37,14 @@ div {
...
 #list-container {
-  border: 2px solid #101010;
+  border: 2px dashed #6edcd9;
   width: 100%;
   font-size: 12px;
 }
 
 li {
   border-bottom: 1px dashed #6edcd9;
+  color: #0c0c0c;
```
Comment (alexander16108) [inline src/style.css:47]: - [x] optional - currently ur button looks like this 
image
but I would suggest that you increase the padding inside the button to reveal the text 👍🏾

File: src/style.css [hunk 2]
```diff
@@ -18,11 +19,13 @@ main {
...
 h1 {
+  color: #1fb0dc;
   font-size: 26px;
   margin: 50px 0 80px 0;
 }
 
 h2 {
+  color: #f2192f;
   font-size: 16px;
 }
```
Comment (alexander16108) [inline src/style.css:31]: - [x] optional - 😸 its a lovely concept of using monochromatic color but i would strong suggest u use more color to style page to improve the UX design of the page

**Review by alexander16108 — APPROVED** [review]
Hi @Melaku05 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (Melaku05) [reply src/style.css:31]: okay, Thank you I will do that.

File: src/style.css (hunk 1 above)
Comment (Melaku05) [reply src/style.css:47]: Thank you, I will fix that.

**Pull request merged**